sites/cheerpx/src/content/docs/13-tutorials/setup-svelte.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Svelte setup
+description: Setting up Svelte app
+---
+
+This tutorial will explain how to create a application with Svelte that works together with CheerpX.
+
+## Install Svelte
+
+First create a new Svelte app by running:
+
+```
+npm create svelte@latest my-app
+```
+
+1. Select the Skeleton Project option.
+2. Enable Typescript if needed.
+3. There are no additional options required for this tutorial.
+
+Next, navigate to your project directory and install the dependencies:
+
+```
+cd my-app
+npm install
+npm run dev
+```
+
+At this point, the development server will run from another terminal in the background.
+
+We're able to test the development server by visiting: http://localhost:5173/
+
+## Install Cheerpx
+
+Next, let's install CheerpX.
+
+This [link] will direct you to the installation page of CheerpX.
+
+For this to work all top level imports used by the CheerpX NPM module need to be enabled.
+
+We'll also need to enable Cross origin isolation. It's required since CheerpX uses SharedArrayBuffer.
+
+Add the following code in vite.config.ts:
```

Comment:
Indentation is inconsistent. Also, people should replace the contents. not add to the content. Being the block with ```ts title=vite.config.ts to clarify both language and file name

Reply:
I am not sure where the indentation is inconsistent.
I've changed the adding with replacing the content.

The reason why I removed the vite.config.ts from the block ``` was because of a failed to parse Markdown issue due to it not existing. That issue was also with the src/routes/+page.ts block.

Reply:
About indendation, see the object passed to `defineConfig`. Notice that optimizeDeps and plugins are not aligned.

Reply:
> The reason why I removed the vite.config.ts from the block ``` was because of a failed to parse Markdown issue due to it not existing

Try to avoid the use of unclear pronoun such as `it`, the syntax works for other files. _what_ is not existing exactly? See the simple executable tutorial for an example.